meta.txt

```diff
@@ -2,6 +2,7 @@ project name, version, upstream URL, upstream commit hash base, license, tests (
 adler32-rs, 1.0.4, https://github.com/remram44/adler32-rs, b235700967618065f89f188a0000d67d08451705, zlib, full
 anyhow, 1.0.28,https://github.com/dtolnay/anyhow, 022f700a1c67c5cd663c34b8a3c11c6818ec0ecb, MIT/Apache-2.0, partial
 cfg-if, 0.1.10, https://github.com/alexcrichton/cfg-if, 4484a6faf816ff8058088ad857b0c6bb2f4b02b2, MIT/Apache-2.0, full
+color_quant, master, 3a1d0c129301adcc1ad4caa91e19089294721c58, MIT, no
```

Comment:
upstream URL? test should be `none`.

Reply:
Added now~ @mssun

Reply:
Test should be one of `none, partial, full, none-no-std`. This is `none`, but not `no`.

Reply:
Updated~